.gitignore

```diff
@@ -30,6 +30,7 @@ R/pkg/tests/fulltests/Rplots.pdf
 build/*.jar
 build/apache-maven*
 build/scala*
+build/zinc*
```

Comment:
This is because we removed the standalone zinc out (because scala-maven-plugin now has built-in support), see also SPARK-34539. Just remove that file. It won't be generated again.

Reply:
Got it. Sorry for reporting this issue. 
I think we could close this PR.